presto-raptor/src/main/java/com/facebook/presto/raptor/storage/OrcStorageManager.java

```diff
@@ -125,6 +126,8 @@
     private static final JsonCodec<ShardDelta> SHARD_DELTA_CODEC = jsonCodec(ShardDelta.class);
 
     private static final long MAX_ROWS = 1_000_000_000;
+    // Do not limit the max size of blocks to read
```

Comment:
Make this a todo to enable this when we work out the behavior in the hive connector